systers/powerup-iOS
# Allow scene to replay after going to map

### Description
Currently the user cannot replay a scene after finishing it and going to the map, which is inconsistent with the Android version. Instead of changing the segue to the Results View Controller and deleting the Completed View Controller, the Completed View Controller has been changed so that it looks like the Results View Controller (in case the Completed View Controller still needs to be used in the future) and allows the user to replay a scene.

Fixes #200

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Steps to Reproduce:
1. Create a new avatar.
2. Finish the home scene.
3. Go to the map.
4. Click the home scene again. The scene can now be replayed.
5. This works with all other scenes as well.

You can watch this video to see it working. https://www.flickr.com/photos/156368229@N03/25953290398/in/dateposted-public/

### Checklist:


**Code/Quality Assurance Only**
